runner/src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -613,6 +616,26 @@ export class PageControllerBase {
         savedState
       );
 
+      if (true) {
```

Comment:
probably dont want the if true, this is where you'd want the feature toggle if you're using one

Reply:
added a config option for turning this off and on